Dockerfile.test.php7

```diff
@@ -1,7 +1,7 @@
-FROM php:7.2
+FROM php:7.2.2-fpm
```

Comment:
The description of this PR doesn't really describe what this is changing and why. Why do we need the -fpm variant of php? Why do we need such a specific version?

Reply:
https://github.com/aces/Loris/pull/4944/files
7 lines, no different.